### PR TITLE
Upgrade versions of catalog (0.0.27) and common (0.0.39)

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Added project display name to common bind service form component.

### CODE REVIEW

File: app/scripts/directives/bindService.js [hunk 1]
```diff
@@ -20,11 +21,13 @@
...
                        $filter,
                        DataService,
-                       BindingService) {
+                       BindingService,
+                       ProjectsService) {
     var ctrl = this;
     var validityWatcher;
     var bindingWatch;
     var statusCondition = $filter('statusCondition');
+    console.dir(ctrl.target);
```
Comment (dtaylor113) [inline app/scripts/directives/bindService.js]: Do we want this in production code?

File: app/scripts/directives/bindService.js [hunk 2]
```diff
@@ -178,10 +180,13 @@
       });
 
       if (ctrl.target.kind === 'Instance') {
-        ctrl.bindType = "secret-only";
-        ctrl.appToBind = null;
-        ctrl.serviceToBind = ctrl.target.metadata.name;
-        loadApplications();
+        ProjectsService.get(ctrl.target.metadata.namespace).then(_.spread(function(project) {
```
Comment (jwforres) [inline app/scripts/directives/bindService.js]: This is going to cause another request for the project when we already have the project loaded

Comment (jeff-phillips-18) [inline app/scripts/directives/bindService.js]: where is the project?

File: app/scripts/directives/bindService.js (hunk 1 above)
Comment (jwforres) [inline app/scripts/directives/bindService.js]: No, needs to be removed

Comment (jeff-phillips-18) [inline app/scripts/directives/bindService.js]: sorry, left over...

File: app/scripts/directives/bindService.js (hunk 2 above)
Comment (jwforres) [inline app/scripts/directives/bindService.js]: You should update the bindService component to take the project as a param, anywhere we are using that component the project has already been loaded by the main controller for the page

Comment (jwforres) [inline app/scripts/directives/bindService.js]: in both list-row and service-instance-row its available as row.state.project

Comment (jeff-phillips-18) [inline app/scripts/directives/bindService.js]: Awesome. Thanks @jwforres !

Comment (jeff-phillips-18) [inline app/scripts/directives/bindService.js]: Updated. So much better. Thanks again.

**Comment by jwforres** [comment]
[merge]

**Pull request merged**